www/squid/src/opnsense/mvc/app/controllers/OPNsense/Proxy/Api/ExportController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ *    Copyright (C) 2024 0xThiebaut [thiebaut.dev]
```

Comment:
I don't mind a github handle but either with real mail address or completely omitting the secondary information